modules/installation-creating-aws-vpc-carrier-gw.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones)
+// * post_installation_configuration/aws-compute-edge-tasks-zones.adoc (AWS zone tasks) 
+
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
+ifeval::["{context}" == "aws-compute-edge-tasks-zones"]
+:post-aws-zones:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-creating-aws-vpc-carrier-gw_{context}"]
+ifdef::wavelength-zone[]
+= Creating a VPC carrier gateway
+endif::wavelength-zone[]
+ifdef::post-aws-zones[]
+= Wavelength Zones only: Creating a VPC carrier gateway
+endif::post-aws-zones[]
+
+To use public subnets, such as deploying load balancers or edge worker nodes, in your {product-title} cluster on {zone-types}, you must create the carrier gateway and associate it to the VPC. Carrier gateways exist for VPCs that only contain subnets in a Wavelength Zone. A carrier gateway supports IPv4 traffic.
```

Comment:
is `zone-types` defined? I saw `{zone-types}` present in [doc page](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/aws-compute-edge-tasks-zones#installation-creating-aws-vpc-carrier-gw_aws-compute-edge-tasks-zones)?

Reply:
Hi @yunjiang29 . The attribute is defined in the following .adoc files (high-level book/assembly files that contain all the sub-sections):

* installing-aws-wavelength-zone
* installing-aws-localzone
* installing-aws-outposts-remote-workers

Example: 

![Screenshot from 2024-02-06 12-22-54](https://github.com/openshift/openshift-docs/assets/57954076/81940569-eed3-4b00-956d-a70791dfa744)

Reply:
@dfitzmau maybe the problem is the plural? `{zone-types} != {zone-type}`

Reply:
I accidently entered "zone-types" instead of "zone-type". I checked each doc so this issue no longer persists.